.github/workflows/build-common.yml

```diff
@@ -194,8 +194,7 @@ jobs:
           cache-read-only: ${{ inputs.cache-read-only }}
 
       - name: Build
-        # javadoc task fails sporadically fetching https://docs.oracle.com/javase/8/docs/api/
-        run: ./gradlew check spdxSbom -x javadoc -x spotlessCheck -PskipTests=true ${{ inputs.no-build-cache && '--no-build-cache' || '' }}
+        run: ./gradlew check spdxSbom -x spotlessCheck -PskipTests=true ${{ inputs.no-build-cache && '--no-build-cache' || '' }}
```

Comment:
Apparently this still doesn't trigger javadoc. I guess it originally used `assemble` instead of `check` when javadoc was disabled. Try adding `javadoc` to the tasks. Alternatively could run javadoc as a separate step that doesn't block merge if it is still flaky.